helm/hpcc/templates/_helpers.tpl

```diff
@@ -825,6 +825,24 @@ A kludge to ensure until the mount of a PVC appears (this can happen with some t
       mountPath: {{ .volumePath | quote }}
 {{- end }}
 
+{{/*
+Inject container to perform any post plane initialization validation
+Pass in dict with volumeName, volumePath and cmds
+*/}}
+{{- define "hpcc.validatePlaneScript" -}}
+- name: {{ printf "validate-plane-script-container-%s" .volumeName }}
```

Comment:
Would it be worth combining this into the wait for mount container if possible?  What is the extra overhead starting a new container?

Reply:
well both are a kludge, but I wouldn't see both being used in tandem.
I was talking to the person behind the TF module that provisions the raided nvme on the nodes, and he thinks that it may require a new CSI behind the persistent volume to correctly wait for the mount to be fully ready.
But in the short term, in the next release, he is adding a marker file in the mount this mechanism could look for.

All less than ideal, once k8s pods see the pvc, that should be it - we shouldn't have to wait or check like this, but I think the provisioning of these raided nvme's is unusual and not fully supported by AKS out of the box.